FastSearch.UnitTests/SearchTests.cs

```diff
@@ -156,5 +156,60 @@ public void IndexAndSearchTest2()
             Assert.AreEqual(r1.Count, r2.Count, list.Count);
             Assert.AreEqual(r1.Count, r2.Count, union.Count);
         }
+        [TestMethod]
+        public void ExactMatchSearch()
+        {
+            var lines = new List<string>
+            {
+                "corperation"
+            };
+
+            var s = new CharSequenceSearch<string>(lines, enableExactMatch : true);
+
+            var result = s.Search("corpERation");

```

Comment:
This is a bad test. For the test to be effective, there should be values in the lines that do not match.